packages/flame/lib/src/effects2/size_effect.dart

```diff
@@ -0,0 +1,46 @@
+import '../../components.dart';
+import '../../extensions.dart';
+import 'component_effect.dart';
+import 'effect_controller.dart';
+
+/// Change the size of a component over time.
+///
+/// The following constructors are provided:
+///
+///   - [SizeEffect.by] will set the size in relation to it's current size;
```

Comment:
`it's` -> `its`.
Also, this could use some more detailed explanation. Is the effect going to take the current size and add some additional size to it? Say, if the current size is `(100,100)` and you do `SizeEffect(10,-10)` -- will it end up having size `(110,90)`?